packages/p2p/src/components/my-profile/payment-methods/add-payment-method/cancel-add-payment-method-modal.jsx

```diff
@@ -72,7 +72,8 @@ const CancelAddPaymentMethodModal = ({ is_floating }) => {
                 <Button
                     large
                     onClick={() => {
-                        my_profile_store.setIsCancelAddPaymentMethodModalOpen(false);
+                        // my_profile_store.setIsCancelAddPaymentMethodModalOpen(false);
```

Comment:
why is this commented out, we can just delete this right

Reply:
Oof this should not be commented out